.travis.yml

```diff
@@ -2,4 +2,16 @@ sudo: false
 language: ruby
 rvm:
   - 2.4.3
-before_install: gem install bundler -v 1.16.1
+install:
+  - sudo apt-get update
+  - sudo apt-get -y install apt-utils automake
+  - wget https://archive.apache.org/dist/pulsar/pulsar-2.4.1/DEB/apache-pulsar-client.deb
+  - wget https://archive.apache.org/dist/pulsar/pulsar-2.4.1/DEB/apache-pulsar-client-dev.deb
+  - sudo chmod 755 ./apache-pulsar-client.deb
```

Comment:
you shouldn't need to change the mode on these files

Reply:
That's right, just removed the commands. Thanks for the finding!